tests/metrics/privacy_risk_score_test.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+import numpy as np
+
+import torch
+import torch.nn as nn
+
+from privacy_evaluator.models.torch.dcti.dcti import load_dcti
+from privacy_evaluator.datasets.cifar10 import CIFAR10
+
+from privacy_evaluator.metrics import compute_privacy_risk_score
+from privacy_evaluator.classifiers.classifier import Classifier
+
+
+def test_privacy_risk_score():
+    x_train, y_train, x_test, y_test = CIFAR10.numpy(model_type="torch")
+    classifier = Classifier(
+        load_dcti(),
+        nb_classes=CIFAR10.N_CLASSES,
+        input_shape=CIFAR10.INPUT_SHAPE,
+        loss=nn.CrossEntropyLoss(reduction="none"),
+    )
+    score = compute_privacy_risk_score(
+        classifier, x_train[:100], y_train[:100], x_test[:100], y_test[:100]
+    )
+    assert bool(score)
```

Comment:
This test does not seem to be meaningful to me. Could you please explain, what you thought here?

Reply:
For now this test just runs the privacy risk score calculation, to check weather the calculation works at all, throws no exceptions and does not return an empty array. I think I said in our last meeting, that I wasn't sure how to meaningfully test the privacy risk score, since our scores weren't very different for train and test data and I think you said you wanted to provide a model that is over fitting so we can do a meaningful test.